[DROOLS-3744] "Value" is not highlighted in the right panel

yesamer opened the pull request:
@kkufova @gitgabrio @danielezonca Can you please review it?

I simply added the property title (`fieldName`) in the `propertyNameElements` list in case the baseClass is a simple object (this means it will be added in any case). `isSimple` method is removed because unused.

https://issues.jboss.org/browse/DROOLS-3744

yesamer commented:
@gitgabrio  According to the bug description: in case of a simple data object added to to the scenario, the property title "value" is not highlighted when selected. The reason of this behaviour is in line 377: if a simple class is handled, only the class is taken. And this is wrong: the field name (value) should be added too.

yesamer commented:
@kkufova Please wait before test it

kkufova commented:
@yesamer, it is still not working:

![bug1](https://user-images.githubusercontent.com/16349113/59187316-7d3b8d00-8b75-11e9-9d98-dda941812b18.png)

yesamer commented:
@kkufova Basically I found the same bug on RULE scenario test. I fixed it in my previous commits. Now, I found the problem on DMN side and fixed it too. I kindly ask you to test the fixes on both RULE / DMN cases. I suggest to test nested facts too, just to be sure. 
Moreover, this PR fix https://issues.jboss.org/browse/DROOLS-4078 as well. Can you please verify it?
Thanks

kkufova commented:
@yesamer, is this ready for another review?

yesamer commented:
@kkufova Not yet :)

danielezonca commented on drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/utils/ScenarioSimulationGridHeaderUtilities.java:
```diff
@@ -151,10 +152,18 @@ public static String getExistingInstances(final String group, final ScenarioGrid
+    public static List<String> getPropertyNameElements(final ScenarioGridModel scenarioGridModel, final int columnIndex) {
+        final Optional<Simulation> optionalSimulation = scenarioGridModel.getSimulation();
+        return optionalSimulation.map(simulation -> {
+            final FactMapping factMapping = scenarioGridModel.getSimulation().get().getSimulationDescriptor().getFactMappingByIndex(columnIndex);
```
You can use `simulation` here instead of access again via `scenarioGridModel.getSimulation()...`

yesamer replied:
@danielezonca Done, thank you!
